docs/dummyvm/src/scanning.rs

```diff
@@ -0,0 +1,40 @@
+use crate::slots::DummyVMSlot;
+use crate::DummyVM;
+use mmtk::util::opaque_pointer::*;
+use mmtk::util::ObjectReference;
+use mmtk::vm::RootsWorkFactory;
+use mmtk::vm::Scanning;
+use mmtk::vm::SlotVisitor;
+use mmtk::Mutator;
+
+pub struct VMScanning {}
+
+// Documentation: https://docs.mmtk.io/api/mmtk/vm/scanning/trait.Scanning.html
+impl Scanning<DummyVM> for VMScanning {
+    fn scan_roots_in_mutator_thread(
+        _tls: VMWorkerThread,
+        _mutator: &'static mut Mutator<DummyVM>,
+        _factory: impl RootsWorkFactory<DummyVMSlot>,
+    ) {
+        unimplemented!()
+    }
+    fn scan_vm_specific_roots(_tls: VMWorkerThread, _factory: impl RootsWorkFactory<DummyVMSlot>) {
+        unimplemented!()
+    }
+    fn scan_object<EV: SlotVisitor<DummyVMSlot>>(
+        _tls: VMWorkerThread,
+        _object: ObjectReference,
+        _edge_visitor: &mut EV,
+    ) {
```

Comment:
We have renamed it to `SlotVisitor`.  The generic parameter and value parameter should be changed, too.

```suggestion
    fn scan_object<SV: SlotVisitor<DummyVMSlot>>(
        _tls: VMWorkerThread,
        _object: ObjectReference,
        _slot_visitor: &mut SV,
    ) {
```